typespec-tests/src/main/java/com/cadl/builtin/implementation/BuiltinClientImpl.java

```diff
@@ -16,12 +16,12 @@
  */
 public final class BuiltinClientImpl {
     /**
-     * Server parameter.
+     * Service host.
```

Comment:
`endpoint` parameter desription change due to TCGC use this description, and I just hornor TCGC's desc.